draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -816,9 +816,11 @@ A mirror maintains a copy of the CA's latest batch number, and batch state. Roug
 
    2. Compute the tree head for the assertion list, as described in {{building-tree}}. If this value does not match the fetched tree head, abort this procedure with an error.
 
-   3. Compute the ValidityWindow structure and verify the signature, as described in {{signing}}. Set `tree_heads[0]` to the tree head fetched above. Set the other values in `tree_heads` to the previously mirrored values. If signature verification fails, abort this procedure with an error.
+   3. Compute the expected ValidityWindow structure as follows: Set `batch_number` to `i`. Set `tree_heads[0]` to the tree head fetched above. Set the other values in `tree_heads` to the previously mirrored values.
```

Comment:
I noticed that this procedure is written as if you construct the ValidityWindow you expect rather than fetching it. I kept that but depending on how the HTTP format shakes out, maybe that'll change.

(Also I'm getting convinced that maybe we should just chain the hashes together. It makes describing signatures more straightforward because you don't need to sign over the whole window, just the last hash. Though we still need to talk about validity windows because clients need to receive the whole hash.)